src/towncrier/_builder.py

```diff
@@ -115,6 +115,8 @@ def find_fragments(
     If strict, raise ClickException if any fragments have an invalid name.
     """
     ignored_files = {".gitignore", ".keep", "readme", "readme.md", "readme.rst"}
+    if isinstance(config.template, str):
```

Comment:
Why we need to do this check ?

Can't we just alwasy add it.

I understand that sometimes this can be `towncrier:default.rst` ... but I think that it's ok to just add that value...

I don't expact that people will not want to ignore a file with than name.

... just to keep it simple  and don't worry too much about edge cases that will never happen in real life :)

Reply:
According to the types, config.template can be `Tuple[str,str]` so that might cause troubles down the line.

Reply:
As long as the test pass, we should not worry about that and try to write the most simple code possible.

No need to write code for imaginary cases for which we don't have a real world use case.

If this check is indeede needed, I think that it needs a comment to explain why we can end up with a tuple

Reply:
I don't know why we can end with a tuple, as I am first time diving into the source code but according to the dataclass of Config:
```python
@dataclasses.dataclass
class Config:
    template: str | tuple[str, str]
```
should I just disregard it?

Reply:
I have applied this patch and I can see that all the tests still pass via `trial src/twisted`

```diff
diff --git a/src/towncrier/_builder.py b/src/towncrier/_builder.py
index dabd64a..78d8bbe 100644
--- a/src/towncrier/_builder.py
+++ b/src/towncrier/_builder.py
@@ -115,8 +115,7 @@ def find_fragments(
     If strict, raise ClickException if any fragments have an invalid name.
     """
     ignored_files = {".gitignore", ".keep", "readme", "readme.md", "readme.rst"}
-    if isinstance(config.template, str):
-        ignored_files.add(config.template)
+    ignored_files.add(config.template)
     if config.ignore:
         ignored_files.update(filename.lower() for filename in config.ignore)
```

Reply:
but it looks like we are missing an automated test.

I can see that settings is set to a tuple when a package is provided

https://github.com/twisted/towncrier/blob/68a7a3b7e2f04a32a2eea670a5ad5c38913ff470/src/towncrier/_settings/load.py#L187-L203

Reply:
I have applied the changed.
I got to say I highly disagree but you're the boss here :)